homeassistant/components/asuswrt/__init__.py

```diff
@@ -59,13 +66,15 @@
 
     api = AsusWrt(
         conf[CONF_HOST],
-        conf.get(CONF_PORT),
+        conf[CONF_PORT],
         conf.get(CONF_PROTOCOL) == "telnet",
         conf[CONF_USERNAME],
         conf.get(CONF_PASSWORD, ""),
         conf.get("ssh_key", conf.get("pub_key", "")),
         conf.get(CONF_MODE),
         conf.get(CONF_REQUIRE_IP),
```

Comment:
Please use `dict[key]` for `CONF_PROTOCOL`, `CONF_MODE` and `CONF_REQUIRE_IP` as well ✌